samples/client/petstore/objc/SwaggerClientTests/Tests/PetApiTest.m

```diff
@@ -75,12 +75,12 @@ - (void) testUpdatePet {
     XCTestExpectation *expectation = [self expectationWithDescription:@"testUpdatePet"];
     SWGPet* pet = [self createPet];
 
-    [api addPetWithCompletionBlock:pet completionHandler:^(NSError *error) {
+    [api addPetWithBody:pet completionHandler:^(NSError *error) {
         if(error) {
             XCTFail(@"got error %@", error);
         }
         else {
-            [api getPetByIdWithCompletionBlock:[NSString stringWithFormat:@"%@",[pet _id]] completionHandler:^(SWGPet *output, NSError *error) {
+            [api getPetByIdWithPetId:[pet _id] completionHandler:^(SWGPet *output, NSError *error) {
```

Comment:
I'm not sure why the conversion to `NSString` was needed in the original code...?